src/com/dakl/Main.java

```diff
@@ -425,6 +426,7 @@ void nextTurn() {
         num_turns = 1;
     }
 
+    // REVIEW : method naming convention
```

Comment:
The java method naming convention is to start with a lower case - e.g. arrayListToArray(...)